avaframe/com1DFA/particleTools.py

```diff
@@ -54,6 +54,7 @@ def initialiseParticlesFromFile(cfg, avaDir):
     particles['nPart'] = len(xPartArray)
     particles['s'] = np.zeros(np.shape(xPartArray))
     particles['l'] = np.zeros(np.shape(xPartArray))
+    particles['idRel'] = np.ones(np.shape(xPartArray))
```

Comment:
so 1 is a particle that flows and 0 a fixed one?

Reply:
so changed to idFlowing